packages/bruno-cli/src/commands/run.js

```diff
@@ -175,6 +182,31 @@ const handler = async function (argv) {
       envVars = getEnvVars(envJson);
     }
 
+    if (envVar) {
+      let processVars;
+      if (typeof envVar === 'string') {
+        processVars = [envVar];
+      } else if (typeof envVar === 'object' && Array.isArray(envVar)) {
+        processVars = envVar;
+      } else {
+        console.error(chalk.red(`overridable environment variables not parsable: use name=value`));
+        return;
+      }
+      if (processVars && Array.isArray(processVars)) {
+        processVars.forEach((value) => {
+          let parts = value.split('=');
```

Comment:
@mirkogolze This way of doing it will fail if the value has a `=` sign which is not uncommon in access tokens

Can you update this code to handle the case.
Something like
```javascript
processVars.forEach((value) => {
  // split the string at the first equals sign
  const match = value.match(/^([^=]+)=(.*)$/);
  if (!match) {
    console.error(
      chalk.red(`Overridable environment variable not correct: use name=value - presented: `) +
        chalk.dim(`${value}`)
    );
    return;
  }
  const name = match[1];
  const val = match[2];
  envVars[name] = val;
});
```